.github/workflows/tests.yml

```diff
@@ -38,8 +65,18 @@ jobs:
             /bin/bash uninstall.sh -f >/dev/null
           fi
 
+      - name: Set up WSL environment
+        if: runner.os == 'windows'
+        shell: "wsl-bash -u root {0}"
+        run: |
+          echo "runner ALL=(ALL) NOPASSWD:ALL" | tee "/etc/sudoers.d/runner"
+          chmod 644 /etc/sudoers.d/runner
+          ln -sf /home/linuxbrew/.linuxbrew/bin/brew /usr/local/bin/brew
```

Comment:
If you don't want to use a custom shell command, you might instead use a script instead of symlink as `/usr/local/bin/brew` that first sets the `PATH` and then calls `brew`.

Reply:
Oh, btw, your try with removing `--profile` did not work because you are not starting a login shell.
I reread part of the bash man page.
rc-files are only executed if a shell is interactive.
profile-files are only executed if a shell is a login shell.
So actually `--noprofile` and `--norc` are more documentary than effective.
So you could write your path manipulation to the profile file and use a custom `wsl-shell-command` without `--noprofile` and instead with `--login` or `-l` which makes the shell a login shell.

Reply:
Or you write your path manipulation to one of the rc-files and use a custom shell command without `--norc` and instead with `-i` which makes it interactive.